### ISSUE TITLE
Is there a way to go to a specific date?

### ISSUE THREAD

**Issue by leodevbro** [Kiarash-Z/react-modern-calendar-datepicker#199]
let's say I click on some button on my website and the `<DatePicker />` (or `<Calendar />`) goes on 2007-12-15 and picks that date, is this possible?

**Comment by thomasmikava** [comment]
Currently it is not possible

**Comment by leodevbro** [comment]
I guess one good workaround is to just re-render the datepicker on specific date, but I don't think it will work when we want to pick range of days.

**Comment by leodevbro** [comment]
My foolish mistake, it can go to a specific date with just setValue function of the useState, it just needs to be a new reference of object {year: 2020, month: 10, day: 28} in order to force rerender.
I changed the day, but forgot that the object reference stays the same, so it was not re-rendering.

**Comment by thomasmikava** [comment]
I thought you only needed going to specific date without selecting it.

**Comment by leodevbro** [comment]
yeah, I wanted to go to specific date programmatically (without mouse/keyboard interaction), for example 3 seconds after mounting.
